src/libcore/ops/drop.rs

```diff
@@ -1,26 +1,26 @@
-/// Used to run some code when a value goes out of scope.
-/// This is sometimes called a 'destructor'.
+/// Destructor for cleaning up resources.
```

Comment:
I think this is still a little inaccurate, `Drop` itself it not the destructor, but a "hook" for destructor. Maybe change it to something like "Custom code within the destructor for cleaning up resources."

Reply:
Changed it to `Custom code within the destructor`. `Drop` is also used for panic guards (which don't feel like a resource to me)